src/erlcloud_aws.erl

```diff
@@ -822,9 +822,19 @@ get_host_vpc_endpoint(Service, Default) when is_binary(Service) ->
     %% resolve through ENV if any
     Endpoints = case ConfiguredEndpoints of
         {env, EnvVarName} when is_list(EnvVarName) ->
-            Es = string_split(os:getenv(EnvVarName, ""), ","),
             % ignore "" env var or ",," cases
-            [list_to_binary(E) || E <- Es, E /= ""];
+            % also handle "zoneID:zoneName" form when it comes from CFN
+            Es = string_split(os:getenv(EnvVarName, ""), ","),
+            lists:filtermap(
+                fun ("") -> false;
+                    (Value) ->
+                        case string_split(Value, ":") of
+                            [_Id, Name] -> {true, list_to_binary(Name)};
+                            [Name] -> {true, list_to_binary(Name)}
```

Comment:
Do we want a default case clause here with `false`? Or throw an exception intentionally?

Reply:
? support for current/old way of just names in the list is still neded

Reply:
I mean `split_string` returns a list fo 3 and more elements

Reply:
case clause is a good exception